src/components/FormComponent/FormComponent.css

```diff
@@ -0,0 +1,58 @@
+.form-style {
+    font: 95% Arial, Helvetica, sans-serif;
+    max-width: 400px;
+    margin: 10px auto;
+    padding: 16px;
+    background: #F7F7F7;
+}
+
+.form-style input[type="text"],
+.form-style input[type="email"],
+.form-style input[type="password"],
+.form-style input[type="checkbox"]
```

Comment:
Please add a separate class for input fields, name for example "formInputField" and then apply these styles to that class, and give each input field value "formInputField" for the class attribute. This is harder to read, takes up more lines of code and would apply to any new input field you add which might not be the behavior you want.

Reply:
What @kim-kolosek said is very important in this case